Cargo.toml

```diff
@@ -213,7 +213,7 @@ tokio-postgres = { version = "0.5.5", optional = true, features = ["runtime", "w
 # Indirect dependency; pinning until
 # https://github.com/timberio/vector/issues/6005 is resolved
 thread_local = "= 1.0.1"
-dashmap = "3"
+dashmap = "4"
```

Comment:
This might affects perf globally as it's used in metrics. We should take a look at the benches.

Reply:
The is regression, thanks for reminding! I've reverted the changes.